.github/pull_request_template.md

```diff
@@ -1,10 +1,20 @@
 ## Proposed change
 
-<!-- Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that is required for this change. -->
+<!--
+Please include a summary of the changes and the related issue.
+Please also include relevant motivation and context.
+-->
 
 ## Related issues
 
-- :bug: Fixes #(issue)
-- :rocket: Feature #(issue)
+<!--
+Please make sure to follow the [contributing guidelines](https://github.com/amadeus-digital/Otter/blob/main/CONTRIBUTING.md)
+-->
 
-<!-- Please make sure to follow the contributing guidelines on https://github.com/amadeus-digital/Otter/blob/main/CONTRIBUTING.md -->
+*- No issue associated -*
+
+[//]:: "- :bug: Fixes #issue "
```

Comment:
It's a way to comment the text.
It can be:
 - `[//]:: "- <message>"
 - `<!-- - <message> -->`

I picked the first one because of the `-` char of the list.

Reply:
Shouldn't we have only 1 way to do comment to avoid confusion ?

Does it needs to be a list ?
most of the time 1 PR will be associated to 1 issue 
if it's not the case maybe it could be splited 😄